.gitlab/hpsf-gitlab-ci.yml

```diff
@@ -3,7 +3,16 @@ NVIDIA-GH200:
   tags: [nvidia-gh200]
   image: masterleinad/kokkos-nvcc:12.6.1
   script:
-    - cmake -B build -DKokkos_ENABLE_CUDA=ON -DKokkos_ARCH_HOPPER90=ON -DKokkos_ENABLE_IMPL_CUDA_UNIFIED_MEMORY=ON -DKokkos_ENABLE_TESTS=ON
+    - cmake
+      -B build
+      -DCMAKE_CXX_COMPILER=`pwd`/bin/nvcc_wrapper
```

Comment:
Why did we change this?

Reply:
Using nvcc-specific `CMAKE_CXX_FLAGS` flags need `CMAKE_CXX_COMPILER` to be `nvcc_wrapper`.